src/config.ts

```diff
@@ -525,16 +525,10 @@ export const ARNS_NAME_LIST_CACHE_HIT_REFRESH_INTERVAL_SECONDS =
     `${60 * 60}`, // 1 hour
   );
 
-export const ARNS_ANT_STATE_CACHE_MISS_REFRESH_INTERVAL_SECONDS =
+export const ARNS_ANT_STATE_CACHE_HIT_REFRESH_BUFFER_SECONDS =
   +env.varOrDefault(
-    'ARNS_ANT_STATE_CACHE_MISS_REFRESH_INTERVAL_SECONDS',
-    `${10}`, // 10 seconds
-  );
-
-export const ARNS_ANT_STATE_CACHE_HIT_REFRESH_INTERVAL_SECONDS =
-  +env.varOrDefault(
-    'ARNS_ANT_STATE_CACHE_HIT_REFRESH_INTERVAL_SECONDS',
-    `${60 * 5}`, // 5 minutes
+    'ARNS_ANT_STATE_CACHE_HIT_REFRESH_BUFFER_SECONDS',
```

Comment:
Still thinking about the name and default value for this. I might reduce it to 30 seconds.

Reply:
a2202a47e1f1262654fec9b92f9dfa1bf522d651